### DISCUSSION TITLE
Livewire updates property collection order somehow

### DISCUSSION THREAD

**Discussion by sasaorasanin** [livewire/livewire#7694] [7. Bugs (archived)] [answered]
I have very simple livewire class Test,  where I set the `$products` wihin mount function. For some strange reason, when I `dd($this->products)` my  collection order is changed to ASC, not sure why. Any help?

I changed to bugs, because I just created new laravel and lates livewire app. And same thing happens.

```php
<?php

namespace App\Livewire;

use App\Models\Shop\Product;
use Livewire\Component;

class Test extends Component
{
    public $products;

    public function mount () {
        $this->products = Product::latest()->get();
    }

    public function addToCart() {
        dd($this->products);
    }

    public function render()
    {
        return view('livewire.test');
    }
}
```
```php
<div>
    @foreach ($this->products as $product)
    <div wire:key="item-{{$product->id}}">
          {{ $product->name }} - 
            <button wire:click="addToCart"  type="button">Test</button>
    </div>
    @endforeach
</div>
```

**Comment by joshhanley** [comment, marked as answer]
@soixt this has been fixed in this PR https://github.com/livewire/livewire/pull/7359#issuecomment-1878142483 and will be available in the next release 🙂

**Reply by sasaorasanin** [reply]
Oh great, I though i was crazy there :D
